### DISCUSSION TITLE
Amending the event booking page

### DISCUSSION THREAD

**Discussion by RuiDom** [alfio-event/alf.io#1106] [General]
Complete newbie here, so looking for basic advice. Is it possible to take the "Sales start" and "Sales ends" info off the page you get to from the event booking URL? I don't think most people bookign on to an event are that bothered about when those change, plus we have a donation line on ours and the dates aren't relevant for that! :-)

**Comment by cbellone** [comment]
Hi @RuiDom ,

thanks for using alf.io :)

it is not possible to modify the HTML of the event page at runtime, however it is possible to inject additional CSS by using the following two options:

![image](https://user-images.githubusercontent.com/3385346/187658893-784a3796-cbad-4a48-887f-dcce6b519634.png)

- `Base custom css` affects the events list page (if you have more than one public events), event detail, and reservation flow
- `Event custom css` affects the event detail and reservation flow, can be overridden per-event so you can have different styling for different events

for example, if you want to get rid of the whole section completely, you can set

```css
div.card-sale-period {
 display: none;
}
```

in the Event custom css option

**Reply by RuiDom** [reply]
Thanks, Celestino. I can use Firefox's web dev tool to try to find the relevant div names, but are these defined anywhere in the documentation? Is it possible to see the base html to see which bits can be restyled?

**Reply by RuiDom** [reply]
That's worked brilliantly and the web dev tool in Firefox has given me what I needed. Thank you!

**Reply by cbellone** [reply]
there's no documentation currently available, and most importantly there's no guarantee that the class names won't change in the future. Of course we'll do our best to maintain compatibility, but we can't guarantee it.

We will soon include the Angular web application in the main alf.io repo. For now you can see it [here](https://github.com/alfio-event/alf.io-public-frontend/tree/2.0-M4-maintenance)

Please don't forget to flag this conversation as "answered" if you solved your issue. Thanks in advance for that

**Reply by RuiDom** [reply]
@cbellone sorry, but I can't see where to mark this as answered. I feel you have answered it, but not sure how to do this.
